hack/ci/wandisco-git.repo

```diff
@@ -0,0 +1,6 @@
+[wandisco-git]
```

Comment:
This is really weird.

Reply:
Is there a better approach (besides using HEREDOC) ?

Reply:
I just meant that we have to update git from somewhere ...

Reply:
Oh, yes, you are right.

Reply:
: ) git in centos7 is old :woman_shrugging: originally you were basing off of fedora and there git version was later.